src/elements/content-sidebar/activity-feed/task-new/TaskActions.js

```diff
@@ -14,35 +14,42 @@ type Props = {|
     taskType?: TaskType,
 |};
 
-const TaskActions = ({ onTaskApproval, onTaskReject, onTaskComplete, taskType }: Props): React.Node => (
-    <div className="bcs-task-pending-assignment bcs-task-assignment-actions">
-        {taskType === TASK_TYPE_APPROVAL ? (
+const TaskActions = ({ onTaskApproval, onTaskReject, onTaskComplete, taskType }: Props): React.Node => {
+    let action = null;
```

Comment:
Previously, all non-`TASK_TYPE_APPROVAL` cases returned the single-button "complete" action. Do we intend to support an empty task or should we short-circuit and return null if the `taskType` isn't recognized?

Reply:
It no longer makes sense to have a fallback value since the enums passed to the update endpoints are strictly bound to the task type.

Reply:
It looks like the `taskType` prop is optional, though. Thoughts about returning null rather than the div wrapper if `taskType` is not provided?

Reply:
I'll change the flow type to make the type non nullable